SU2_CFD/src/numerics/flow/convection/roe.cpp

```diff
@@ -224,6 +311,11 @@ CNumerics::ResidualType<> CUpwRoeBase_Flow::ComputeResidual(const CConfig* confi
   if (implicit) {
     GetInviscidProjJac(Velocity_i, &Energy_i, Normal, 0.5, Jacobian_i);
     GetInviscidProjJac(Velocity_j, &Energy_j, Normal, 0.5, Jacobian_j);
+
+    if (muscl) {
+      GetMUSCLJac(muscl_kappa, Jacobian_i, Limiter_i, Limiter_j, Velocity_i, &Density_i);
```

Comment:
In defining this boolean in the constructor it would make sense to check the value of MUSCL kappa to check if any work needs to be done.
And possible to include USE_ACCURATE_FLUX_JACOBIANS also in that boolean.